client/src/pages/addtree/TreeInfo.js

```diff
@@ -12,21 +13,24 @@ import Widget from '../../components/Widget';
 import ErrorMessageAll from '../error/ErrorPage';
 import { treeHealth } from '../../util/treeHealth';
 
-// Create an array of DBH menu items: <1, 1 - 24, 30, 36, 42, 48, 56 inches.
-const dbhItems = [
-  <MenuItem key="<1 inch" value="<1 inch">less than 1"</MenuItem>
-].concat(
-  [...Array(25).keys()]
-    .slice(1)
-    .concat(30, 36, 42, 48, 56)
-    .map(value => <MenuItem key={value} value={value}>{value}"</MenuItem>)
-);
+// Create an array of DBH/height menu items: [.25,.5,.75,1,2,3,4] inches.
+const treeSmallValues = [0.25, 0.50, 0.75];
+const treeLargeValues = [...Array(452).keys()];
```

Comment:
Should the here range really be `[1..451]` for a newly planted tree?

Reply:
I think I saw that documentary, about the Jungle Cruise? 

Google says the tallest tree in the world is 380ft, so 451ft is probably overkill...

Instead of a menu, just have a numeric field where people can enter the number directly?